.github/workflows/nightly-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Nightly Tests
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  tests:
+    name: Tests
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+    uses: ./.github/workflows/tests.yml
+    with:
+      python-version: ${{ matrix.python-version }}
+
+  tests-indy:
+    name: Tests (Indy)
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+
+    uses: ./.github/workflows/tests-indy.yml
+    with:
+      python-version: ${{ matrix.python-version }}
+      indy-version: "1.16.0"
```

Comment:
Some of the nightly tests are failing as expected without the added support for the additional versions of Python.  Both sets of tests are failing with python 3.7 and 3.10 in a similar fashion.  The "standard" tests are failing for python 3.6 because it is not available for ubuntu:latest, only 18.04 and 20.04.

Refer to https://github.com/WadeBarnes/aries-cloudagent-python/actions/runs/3517241353 for and example.

Reply:
Ah, thanks for pointing this out. I threw python 3.6 back into the mix last minute and didn't consider that it might be unavailable. I'll see if I can add ubuntu version as another matrix parameter to solve the issue. When we've officially moved away from python 3.6, I imagine we can just go back to running with ubuntu:latest.